### PR TITLE
Release v0.38.0-rc1

### PR DESCRIPTION
**Pull request by willdavsmith**
# Description

* Release v0.38.0-rc1

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

### CODE REVIEW

File: bicepconfig.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		"dynamicTypeLoading": true
 	},
 	"extensions": {
-		"radius": "br:biceptypes.azurecr.io/radius:latest",
-		"aws": "br:biceptypes.azurecr.io/aws:latest"
+		"radius": "br:biceptypes.azurecr.io/radius:v0.38.0-rc1",
+		"aws": "br:biceptypes.azurecr.io/aws:v0.38.0-rc1"
```
Comment (willdavsmith) [inline bicepconfig.json:9]: @sk593 is this the right PR to do this in? do you forsee any issues?

Comment (sk593) [reply bicepconfig.json:9]: no, this doesnt get updated. the version that lives in the repo will always be latest to correspond with the latest version of radius

Comment (sk593) [reply bicepconfig.json:9]: we auto-generate the bicepconfig during app setup, so that's when this would get populated with a specific release version

Comment (willdavsmith) [reply bicepconfig.json:9]: kk, removing then

**Pull request closed without merging**